pictor/config.py.example

```diff
@@ -36,3 +36,6 @@ AWS = (
   "987654321",
   "rac-iiif",
 )
+
+# baseurl
+IMAGESERVER = "https://iiif.rockarch.org"
```

Comment:
I think this config name could be improved - if it's a URL let's called it IMAGESERVER_URL.